# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/AudioStream.cpp
# Commit: e466266339be
# Full Hash: e466266339beeacbfca6c8c781dcdc02699cac03
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-24 21:44:48
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Don't start with an AudioSink if muted from the start. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141990
# ==============================================================================

diff -r 3ee6cb0034c5 -r e466266339be dom/media/AudioStream.cpp
--- a/dom/media/AudioStream.cpp	Tue May 24 13:09:07 2022 +0000
+++ b/dom/media/AudioStream.cpp	Tue May 24 13:09:07 2022 +0000
@@ -149,8 +149,7 @@
       mPlaybackComplete(false),
       mPlaybackRate(1.0f),
       mPreservesPitch(true),
-      mCallbacksStarted(false)
-    {}
+      mCallbacksStarted(false) {}
 
 AudioStream::~AudioStream() {
   LOG("deleted, state %d", mState.load());
@@ -233,7 +232,8 @@
   auto startTime = TimeStamp::Now();
   TRACE("AudioStream::Init");
 
-  LOG("%s channels: %d, rate: %d", __FUNCTION__, mOutChannels, mAudioClock.GetInputRate());
+  LOG("%s channels: %d, rate: %d", __FUNCTION__, mOutChannels,
+      mAudioClock.GetInputRate());
 
   mSinkInfo = aSinkInfo;
 